gdk4-wayland/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rust-version = "1.64"
 [features]
 v4_4 = ["ffi/v4_4"]
 v4_10 = ["ffi/v4_10", "v4_4"]
+v4_12 = ["ffi/v4_12", "v4_10"]
```

Comment:
Here and elsewhere, does it maybe need `gtk/v4_12` or so too? I.e. is enabling this feature requiring newly added APIs in dependencies?

Reply:
Also why do you only add it for `gdk4-wayland/Cargo.toml` and not all the other crates?

Reply:
> Here and elsewhere, does it maybe need gtk/v4_12 or so too? I.e. is enabling this feature requiring newly added APIs in dependencies?

That is true. I have enabled such features for gdk, there is no gtk dependency involved here.

> Also why do you only add it for gdk4-wayland/Cargo.toml and not all the other crates?

Other crates have that already.